sfaira/data/dataloaders/loaders/d10_1038_s41591_020_1061_7/human_blood_2020_10xsequencing_deng.yaml

```diff
@@ -0,0 +1,76 @@
+dataset_structure:
+     dataset_index: 1
+     sample_fns:
+        - "GSM4579891_ac01_bc_matrix.tar.gz"
+        - "GSM4579892_ac02_bc_matrix.tar.gz"
+        - "GSM4579893_ac03_bc_matrix.tar.gz"
+        - "GSM4579894_ac04_bc_matrix.tar.gz"
+        - "GSM4579895_ac05_bc_matrix.tar.gz"
+        - "GSM4579896_ac06_bc_matrix.tar.gz"
+        - "GSM4579897_ac07_bc_matrix.tar.gz"
+        - "GSM4579898_ac08_bc_matrix.tar.gz"
+        - "GSM4579899_ac09_bc_matrix.tar.gz"
+        - "GSM4579900_ac10_bc_matrix.tar.gz"
+        - "GSM4579901_ac11_bc_matrix.tar.gz"
+        - "GSM4579902_ac12_bc_matrix.tar.gz"
+        - "GSM4579903_ac13_bc_matrix.tar.gz"
+        - "GSM4579904_ac14_bc_matrix.tar.gz"
+        - "GSM4579905_ac15_bc_matrix.tar.gz"
+        - "GSM4579906_ac16_bc_matrix.tar.gz"
+        - "GSM4579907_ac17_bc_matrix.tar.gz"
+        - "GSM4579908_ac18_bc_matrix.tar.gz"
+        - "GSM4579909_ac19_bc_matrix.tar.gz"
+        - "GSM4579910_ac20_bc_matrix.tar.gz"
+        - "GSM4579911_ac21_bc_matrix.tar.gz"
+        - "GSM4579912_ac22_bc_matrix.tar.gz"
+        - "GSM4579913_ac23_bc_matrix.tar.gz"
+        - "GSM4579914_ac24_bc_matrix.tar.gz" 
+    
+dataset_wise:
+    author: "Deng"
+    default_embedding: 
+    doi_preprint: 
+    doi_journal: "10.1038/s41591-020-1061-7"
+    download_url_data: "https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE151511&format=file"
+    download_url_meta: "https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-1061-7/MediaObjects/41591_2020_1061_MOESM3_ESM.xlsx"
+    primary_data: True
+    normalization: "raw"
+    year: 2020
+dataset_or_observation_wise:
+    assay_sc: "10x 5' transcription profiling"
+    assay_sc_obs_key:
+    assay_differentiation:
+    assay_differentiation_obs_key: 
+    assay_type_differentiation:
+    assay_type_differentiation_obs_key:
+    bio_sample:
+    bio_sample_obs_key: 
+    cell_line:
+    cell_line_obs_key:
+    cell_type:
```

Comment:
This is T cell for all cells right? We can add this under `cell_type:`

Reply:
yes, it's CAR T cells for all cells, so just the coarsest ontology?

Reply:
Yes, unless they specify that they only use Cd8 or Cd4 or so.